tfe.go

```diff
@@ -441,6 +441,9 @@ func (c *Client) getRawAPIMetadata() (rawAPIMetadata, error) {
 	req.Header.Set("Accept", "application/vnd.api+json")
 	req.Header.Set("Authorization", "Bearer "+c.token)
 
+	// Temporarily enable retryServerErrors in case the metadata request fails
+	c.retryServerErrors = true
+
 	// Make a single request to retrieve the rate limit headers.
 	resp, err := c.http.HTTPClient.Do(req)
 	if err != nil {
```

Comment:
Thank you for this change.

If it errors here, then it looks like it could get permanently stuck with the retry enabled. I don't know if `getRawAPIMetadata()` will be called somewhere else in the code in the future, but it could result in unexpected side-effects as the codebase grows.

Reply:
I considered this initially but two things jumped out at me that made me hesitate:
- This method is not exported. 
- This method is only called once in go-tfe and that's in `NewClient()`. The user can only modify `c.retryServerErrors` using the exported receiver method `c.RetryServerErrors(bool)` and so we wouldn't be altering any set values since the client has not yet been created.